manager/src/main/java/com/bakdata/quick/manager/graphql/GraphQLToProtobufConverter.java

```diff
@@ -0,0 +1,326 @@
+        final Builder currentMessage,
+        final GraphQLFieldDefinition graphQLFieldDefinition,
+        final GraphQLType graphQLType,
+        final int fieldNumber,
+        final Label label) {
+        if (graphQLType instanceof GraphQLObjectType) {
+            handleObjectType(fileBuilder, currentMessage, graphQLFieldDefinition, fieldNumber, label,
+                (GraphQLObjectType) graphQLType);
+
+        } else if (isScalar(graphQLType)) {
+            final FieldDescriptorProto scalarFieldBuilder =
+                createFieldDescriptor((GraphQLScalarType) graphQLType,
+                    graphQLFieldDefinition.getName(),
+                    fieldNumber,
+                    label);
+            currentMessage.addField(scalarFieldBuilder);
+
+        } else if (isEnum(graphQLType)) {
+            handleEnumType(fileBuilder, currentMessage, graphQLFieldDefinition, fieldNumber, label,
+                (GraphQLEnumType) graphQLType);
+        } else if (isList(graphQLType)) {
+            final GraphQLList graphQLList = (GraphQLList) graphQLType;
+
+            createMessage(fileBuilder,
+                currentMessage,
+                graphQLFieldDefinition,
+                graphQLList.getWrappedType(),
+                fieldNumber,
+                Label.LABEL_REPEATED);
+        } else {
+            throw new BadArgumentException(
+                String.format("Type %s not recognized", GraphQLTypeUtil.simplePrint(graphQLType)));
+        }
+    }
+
+    private static void handleObjectType(
+        final FileDescriptorProto.Builder fileBuilder,
+        final Builder currentMessage,
+        final GraphQLFieldDefinition graphQLFieldDefinition,
+        final int fieldNumber,
+        final Label label,
+        final GraphQLObjectType graphQLObjectType) {
+        currentMessage.addField(buildFieldWithType(graphQLFieldDefinition.getName(),
+            fieldNumber,
+            Type.TYPE_MESSAGE,
+            graphQLObjectType.getName(),
+            label));
+
+        final DescriptorProto.Builder descriptorProtoBuilder = DescriptorProto.newBuilder();
+        descriptorProtoBuilder.setName(graphQLObjectType.getName());
+
+        generateFile(graphQLObjectType.getName(), graphQLObjectType.getFieldDefinitions(), fileBuilder);
+    }
+
+    /**
+     * Creates a FieldDescriptorProto object for a specific type. These fields could be either a type of
+     * TYPE_MESSAGE or TYPE_ENUM. If a FieldDescriptorProto object type is one of these the property TypeName must be
+     * specified otherwise a {@link DescriptorValidationException} is thrown.
+     *
+     * @see com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Type#TYPE_MESSAGE
+     * @see com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Type#TYPE_ENUM
+     * @see com.google.protobuf.DescriptorProtos.FieldDescriptorProto.Builder#setTypeName(String)
+     */
+    private static FieldDescriptorProto buildFieldWithType(
```

Comment:
```suggestion
    private static FieldDescriptorProto createFieldWithType(
```

All the methods returning a proto element start with `create...`, any reason this doesn't?